pkg/webui/console/containers/schedule-delay-input/schedule-delay-input.styl

```
@@ -0,0 +1,43 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.container
+  display: flex
+  justify-content space-between
+  flex-wrap: wrap
+
+  .select
+    flex-basis: 100%
+
+  .number
+    margin-bottom: $cs.xs
+    flex-basis: 100%
+    width: 90%
+
+.notification
+  width: 100%
+
++media-query-min($bp.s)
+  .container
+    flex-wrap: nowrap
+
+    .select
+      flex-basis: 30%
+      margin-bottom: 0
+
+    .number
+      padding-right 2rem
```

```suggestion
      padding-right $cs.s
```
Always use scale variables.